src/components/home/Installer.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '../Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBoxOpen } from '@fortawesome/free-solid-svg-icons';
+// TODO: Update image when installer redesign is finished.
+import InstallerImage from '../../assets/img/InstallerScreenshot.png';
+
+export const Installer: React.FC = () => {
+    return (
+        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark ">
```

Comment:
```suggestion
        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark">
```
Remove additional space.